README.md

```diff
@@ -7,45 +7,681 @@
+> 	 Estimated piece size is 256 bytes.
+> 	 Proposing deal to miner t01459 with 0 fil per epoch...
+> 	 Proposing deal to miner t0117734 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0120993 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0120642 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0121477 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0119390 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0101180 with 10000000 fil per epoch...
+> 	 Proposing deal to miner t0117803 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0121852 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0119822 with 500000000 fil per epoch...
+> 	 Watching deals unfold...
+> 	 Deal with miner t0117803 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121852 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121477 changed state to StorageDealClientFunding
+> 	 Deal with miner t0101180 changed state to StorageDealClientFunding
+> 	 Deal with miner t0119822 changed state to StorageDealClientFunding
+> 	 Deal with miner t0119390 changed state to StorageDealClientFunding
+> 	 Deal with miner t0120642 changed state to StorageDealClientFunding
+> 	 Deal with miner t0117734 changed state to StorageDealClientFunding
+> 	 Deal with miner t01459 changed state to StorageDealClientFunding
+> 	 Deal with miner t0120993 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121477 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0119822 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0117734 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0121852 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t01459 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0120642 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0120993 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0117803 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0101180 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0119390 changed state to StorageDealWaitingForDataRequest
+>    Deal with miner t01459 changed state to StorageDealProposalAccepted
+> 	 Deal with miner t01459 changed state to StorageDealSealing
+```
+
+The output will looks something like the above. With a little luck, you will start seeing some successful storage deals.
+
+To-do: Add section on recovering a bucket from an archive using the Lotus client.
+
+### Multi-writer buckets
+
+Multi-writer buckets leverage the distributed nature of ThreadDB by allowing multiple identities to write to the same bucket hosted by different Libp2p hosts. Since buckets are ThreadDB collection _instances_, this is no different than normal ThreadDB peer collaboration.
+
+To-do: Demonstrate joining a bucket from a ThreadDB invite.
```

Comment:
N: Maybe adding some small comment that mulitple-writers should be responsible for overriding unknown concurrent writes? Might not worth to comment that since is a general problem... but multi-writers open the door to a big set of possible problems if used incorrectly.

Reply:
I'll circle back to this section